### PR TITLE
Add sentry integration

### PR DESCRIPTION
**Pull request by jos-b**
Integrates Sentry with Snekbox.

Adds the FalconIntegration as well to handle Internal Server errors.

Note: This could not be tested locally because Docker & Pipenv on MacOS are hyper-broken for me.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ networkx = "~= 2.4"
 beautifulsoup4 = "~= 4.8"
 pyyaml = "~= 5.2"
 toml = "~= 0.10"
+sentry-sdk = {extras = ["falcon"], version = "0.14.2"}
```
Comment (MarkKoz) [inline Pipfile]: This looks incorrect, but haven't tested. Either way it's a good idea to pin it a bit more loosely.

```suggestion
sentry-sdk = {extras = ["falcon"], version = "~= 0.14.2"}
```

**Comment by jos-b** [comment]
```
pipenv is broken.

the world is crumbling.
```

**Comment by MarkKoz** [comment]
Will someone else need to lock and push it?

**Comment by jos-b** [comment]
I'm locking it now, I fixed pipenv by deleting all my `.DS_Store` files since pipenv cannot handle it.

**Pull request merged**